function_scripts.md

```diff
@@ -23,14 +25,14 @@ Place them on SD card in folder /SCRIPTS/FUNCTIONS/
 
 ##### Lifetime
 
-* script *init* function is called
-* script *run* function is periodically called (inside GUI thread, period cca 30ms) until it returns a non-zero result
+* script *init* function is called once with model is loaded
+* script *run* function is periodically called as long as switch condition is true
 * script is stopped and disabled if it misbehaves (too long runtime, error in code, low memory)
-* all mix scripts are stopped while function script is running 
```

Comment:
all mix and function scripts are stopped while the stand-alone script is running